library/include/modules/Buildings.h

```diff
@@ -141,7 +141,9 @@ DFHACK_EXPORT bool getCorrectSize(df::coord2d &size, df::coord2d &center,
  */
 DFHACK_EXPORT bool checkFreeTiles(df::coord pos, df::coord2d size,
                                   df::building_extents *ext = NULL,
-                                  bool create_ext = false, bool allow_occupied = false);
+                                  bool create_ext = false,
+                                  bool allow_occupied = false,
+                                  bool is_civzone = false);
```

Comment:
Would `allow_wall` be a better name, in case other building types run into this?

Reply:
I wasn't sure how general this would be, but that works for me. The specific call that it's bypassing is HighPassable(). Is that close enough to "isWallI()"?

Reply:
I had to look it up, and I'm still a bit confused what "high passable" means - the comment in the XML file for the passable_high attribute is "tile is missing a roof", which I guess means the "main" part of the tile (not the floor) is open?

I would lean towards a more generic name in this case, and `allow_wall` is (IMO) more generic than `is_civzone`.

Reply:
agreed. done.